karma.conf.js

```diff
@@ -1,40 +1,7 @@
 'use strict';
 
+var karmaConfig = require('./karma.config.shared');
```

Comment:
That was deliberate but clearly not distinct enough. I've moved it to karma.shared.js.